lost_coords: {
    x:          2,
    y:          4,
    z:          0,
    roll:       0,
    pitch:      0,
    yaw:        0,
    w:          1,
}

r1_goal1: {
    x:          -1,
    y:          0,
    z:          0,
    roll:       0,
    pitch:      0,
    yaw:        1.5,
    w:          1,
}

r1_goal2: {
    x:          -1,
    y:          4,
    z:          0,
    roll:       0,
    pitch:      0,
    yaw:        0,
    w:          1,
}

r3_goal1: {
    x:          1,
    y:          -4,
    z:          0,
    roll:       0,
    pitch:      0,
    yaw:        0,
    w:          1,
}

r3_goal2: {
    x:          1,
    y:          0,
    z:          0,
    roll:       0,
    pitch:      0,
    yaw:        0,
    w:          1,
}
